modules/graalvm/22.0.0-java17/module.yaml

```diff
@@ -0,0 +1,34 @@
+schema_version: 1
+name: graalvm
+version: &version "22.0.0-java17"
```

Comment:
Don't we need to reflect the patch number in the version as well? I.e. shouldn't it be `22.0.0.2-java17`?

Reply:
I would keep it that way for now and see how things evolve on their side.

Reply:
Yes, I didn't want to introduce the patch number. We will see if it becomes an essential part of their versioning schema.